### DISCUSSION TITLE
What is the goal of this project in terms of adoption?

### DISCUSSION THREAD

**Discussion by dkogan** [uutils/coreutils#4358] [General]
Hi. Not sure where to ask this, so I'm creating an issue. Is the hope to replace the GNU coreutils in Debian someday? Are you going for feature-parity only, or are you also adding new features to the various tools?

Thanks

**Comment by sylvestre** [comment]
Let's see where it goes. Debian being conservative, it is probably going to be one of the last distro to switch.

The goal is to have feature parity - we don't have it.

We already have some extensions: https://github.com/uutils/coreutils/blob/HEAD/docs/src/extensions.md

**Comment by lengyijun** [comment]
Is it possible to add a feature to `cp`: skip gitignored files ?

**Comment by PThorpe92** [comment]
> Is it possible to add a feature to `cp`: skip gitignored files ?

I would be interested in implementing this, if this is something the project is interested in having.

**Reply by sylvestre** [reply]
Could you please create a new discussion for this ?

**Comment by cnoam** [comment]
New to the discussion so maybe missed something.
Why is this project needed? Why not use GNU coreutils? it is written in C so compiles on any platform.

License? source code safety? dependencies?

**Reply by sylvestre** [reply]
I did a talk last year about this:
https://www.youtube.com/watch?v=90Q5N1qT7BQ

See this slide:
https://sylvestre.ledru.info/presentations/coreutils-fosdem-2023/#25
